examples/tutorials/tutorial_2D_30p30n.py

```diff
@@ -0,0 +1,140 @@
+import flow360 as fl
+from flow360.component.simulation.unit_system import SI_unit_system, u
+from flow360.examples import Tutorial2D30p30n
+
+fl.Env.preprod.active()
+
+project = fl.Project.from_file(Tutorial2D30p30n.geometry, name="Tutorial 2D 30p30n from Python")
+geometry = project.geometry
+
+# show face and edge groupings
+geometry.show_available_groupings(verbose_mode=True)
+geometry.group_faces_by_tag("faceName")
+geometry.group_edges_by_tag("edgeName")
+
+
+with SI_unit_system:
+    cylinders = [
+        fl.Cylinder(
+            name=f"cylinder{i+1}",
+            axis=[0, 1, 0],
+            center=[0.7, 0.5, 0],
+            outer_radius=outer_radius,
+            height=1.0,
+        )
+        for i, outer_radius in enumerate([1.1, 2.2, 3.3, 4.5])
+    ]
+    cylinder5 = fl.Cylinder(
+        name="cylinder5", axis=[-1, 0, 0], center=[6.5, 0.5, 0], outer_radius=6.5, height=10
+    )
+    farfield = fl.AutomatedFarfield(name="farfield", method="quasi-3d")
+    params = fl.SimulationParams(
+        meshing=fl.MeshingParams(
+            defaults=fl.MeshingDefaults(
+                surface_edge_growth_rate=1.17,
+                surface_max_edge_length=1.1,
+                curvature_resolution_angle=12 * u.deg,
+                boundary_layer_growth_rate=1.17,
+                boundary_layer_first_layer_thickness=4.9536767e-06,
+            ),
+            refinement_factor=1.35,
+            gap_treatment_strength=0.5,
+            volume_zones=[farfield],
+            refinements=[
+                fl.UniformRefinement(name="refinement1", spacing=0.1, entities=[cylinders[0]]),
+                fl.UniformRefinement(name="refinement2", spacing=0.15, entities=[cylinders[1]]),
+                fl.UniformRefinement(name="refinement3", spacing=0.225, entities=[cylinders[2]]),
+                fl.UniformRefinement(name="refinement4", spacing=0.275, entities=[cylinders[3]]),
+                fl.UniformRefinement(name="refinement5", spacing=0.325, entities=[cylinder5]),
+                fl.SurfaceRefinement(name="wing", max_edge_length=0.74, faces=[geometry["wing"]]),
+                fl.SurfaceRefinement(
+                    name="flap-slat",
+                    max_edge_length=0.55,
+                    faces=[geometry["flap"], geometry["slat"]],
+                ),
+                fl.SurfaceRefinement(
+                    name="trailing",
+                    max_edge_length=0.36,
+                    faces=[
+                        geometry["wingTrailing"],
+                        geometry["flapTrailing"],
+                        geometry["slatTrailing"],
+                    ],
+                ),
+                fl.SurfaceEdgeRefinement(
+                    name="edges",
+                    method=fl.HeightBasedRefinement(value=0.0007),
+                    edges=[
+                        geometry["wingtrailingEdge"],
```

Comment:
Same suggestion for the enetity selection here, and please also check the other places.

Reply:
👍